src/components/Home/header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./_header.scss";
+import avatar from "../../../assets/images/pngwing.com.png";
+
+import { FaBars } from "react-icons/fa";
+import { AiOutlineSearch } from "react-icons/ai";
+import { MdNotifications, MdApps } from "react-icons/md";
+
+const header = () => {
```

Comment:
component should be in upper case±!!!!!!!!

Reply:
changed component name